hbase-server/src/main/jamon/org/apache/hadoop/hbase/tmpl/master/MasterStatusTmpl.jamon

```diff
@@ -223,7 +223,7 @@ AssignmentManager assignmentManager = master.getAssignmentManager();
         <%if master.getAssignmentManager() != null %>
           <& AssignmentManagerStatusTmpl; assignmentManager=master.getAssignmentManager()&>
         </%if>
-        <%if !master.isInMaintenanceMode() %>
+        <%if !master.isInMaintenanceMode() &&  master.getMasterCoprocessorHost() != null %>
```

Comment:
lgtm

nit: double space between && and 'master'

Reply:
thanks for reminding